_layouts/base.html

```diff
@@ -35,7 +24,10 @@
   </div>
   {% include project-footer.html %}
   {% include redhat-footer.html %}
-    
+  
+  {% if site.google_analytics %}
```

Comment:
So this would _always_ add this, right? Whereas the snippet shouldn't be included when running local for dev, or on staging in the future. Looks like there a notion of "environments" or similar in Jekyll, which we could query here. Look for "jekyll.environment == 'production'" in https://desiredpersona.com/google-analytics-jekyll/.

Reply:
> Good start, one comment on the environment issue, @uidoyen.

Added it for production environment.